src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -10251,7 +10251,7 @@ namespace SurfaceGeometry {
                 WindowShadingControl(a).SetPoint == WindowShadingControl(b).SetPoint &&
                 WindowShadingControl(a).ShadingControlIsScheduled == WindowShadingControl(b).ShadingControlIsScheduled &&
                 WindowShadingControl(a).GlareControlIsActive == WindowShadingControl(b).GlareControlIsActive &&
-                WindowShadingControl(a).SlatAngleControlForBlinds == WindowShadingControl(b).SlatAngleControlForBlinds &&
```

Comment:
`auto &shadingControlA` and `auto &shadingControlB` are more helpful shortcuts than an array shortcut.

Reply:
I don't know why the code did not update here, but do [these changes](https://github.com/NREL/EnergyPlus/blob/609d5a13407106c3f6da617e85e467f0a6207921/src/EnergyPlus/SurfaceGeometry.cc#L10223-L10236) look good?